<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 95, 95);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_agence">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(87, 126, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_employe">
     <attribute name="title">
      <string>Employés</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_client">
     <attribute name="title">
      <string>Clients</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_agence">
     <attribute name="title">
      <string>Agences</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_metier2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_metier6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_metier3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_emplye">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>731</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 170, 72);</string>
      </property>
      <property name="title">
       <string>Employé</string>
      </property>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de l'agence</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Budget</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_budget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adresse">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>356</x>
         <y>30</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre d'employés :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbr_employe">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>356</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de clients :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbr_client">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>356</x>
         <y>110</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de biens :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbr_bien">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_metier5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_metier1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_metier4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Metier 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>651</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimer">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>350</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>350</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID à supprimer :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_bien">
     <attribute name="title">
      <string>Biens</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_reclamation">
     <attribute name="title">
      <string>Réclamations</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
